test/unit/util.js

```diff
@@ -0,0 +1,34 @@
+const resemble = window.resemble;
+
+function getImageBlob(path, callback) {
+	const xhr = new XMLHttpRequest();
+
+	xhr.open("GET", path, true);
+	if (window.URL) {
+		xhr.responseType = "blob";
+	} else {
+		xhr.responseType = "arraybuffer";
+	}
+	xhr.onreadystatechange = function(e) {
+		if (xhr.readyState !== 4 || xhr.status !== 200) {
+			return;
+		}
+		callback(this.response);
+	};
+	xhr.send();
+}
+
+function compare(path, canvas, callback) {
+	getImageBlob(path, reference => {
+		canvas.toBlob(canvasData => {
+			resemble(reference)
+				.compareTo(canvasData)
+				.onComplete(data => {
+					const pct = parseFloat(data.misMatchPercentage);
+
+					callback(pct);
+				}
+			);		
+		}, "image/png");
```

Comment:
Only png available?

Reply:
The format of the test image to be compared is the png format. So we need to extract image binary in png format. jpg format is not used because it is lossy format.